tests/utils/test_downstream.py

```diff
@@ -59,10 +58,10 @@ def test_get_dataset_downstream_dependencies():
         "minimal_mastering_unified_dataset_dedup_clusters_with_data",
         "minimal_mastering_unified_dataset_dedup_signals",
     }
-    assert PIPELINE_DEPENDENT_DATASETS.issubset(set([d.name for d in downstream_datasets]))
+    assert PIPELINE_DEPENDENT_DATASETS == set([d.name for d in downstream_datasets])
```

Comment:
(no change needed) FYI you can do this a set compression: {d.name for d in downstream_datasets}

Reply:
ah that's right, thanks for the suggestion!